### DISCUSSION TITLE
How do you define FOSS?

### DISCUSSION THREAD

**Discussion by farfromrefug** [Hamza417/Inure#349] [Q&A] [answered]
Hi,
I am an app developer. I have a repo which holds 2 apps https://github.com/Akylas/OSS-DocumentScanner
A user just reported that your app report one of the apps as FOSS and not the other one.
So the question is how do you determine if an app is FOSS and its license?

Thanks

**Comment by Hamza417** [comment]
Hey, you can look [here](https://github.com/Hamza417/Inure#include-your-app-in-inures-foss-panel) for the info about this.

**Reply by farfromrefug** [reply]
@Hamza417 thanks for the info. I dont get how my first app OSS Document Scanner got marked as FOSS. I am not on f-droid, only izzydroid. Do you get the info from there too?

**Reply by Hamza417** [reply, marked as answer]
Yes, I take the info from both repos, that's probably why one of your apps got listed under the FOSS category.

**Reply by farfromrefug** [reply]
@Hamza417 ok so i guess i just need to wait for the other one to get picked up ! thanks
